x-pack/plugins/reporting/server/config/create_config.test.ts

```diff
@@ -5,47 +5,22 @@
  * 2.0.
  */
 
-import * as Rx from 'rxjs';
 import { CoreSetup, PluginInitializerContext } from 'src/core/server';
+import { coreMock } from 'src/core/server/mocks';
 import { LevelLogger } from '../lib';
+import { createMockConfigSchema } from '../test_helpers';
 import { createConfig$ } from './create_config';
-import { ReportingConfigType } from './schema';
-
-interface KibanaServer {
-  hostname?: string;
-  port?: number;
-  protocol?: string;
-}
-
-const makeMockInitContext = (config: {
```

Comment:
This change isn't very related to this PR, but this helper code now uses `coreMock`